src/systemctl/systemctl.c

```diff
@@ -7773,6 +7887,8 @@ static int systemctl_help(void) {
                "  daemon-reexec                       Reexecute systemd manager\n"
                "  log-level [LEVEL]                   Get/set logging threshold for manager\n"
                "  log-target [TARGET]                 Get/set logging target for manager\n"
+               "  service-log-level SERVICE [LEVEL]   Get/set logging threshold for service\n"
+               "  service-log-target SERVICE [TARGET] Get/set logging target for service\n"
                "  service-watchdogs [BOOL]            Get/set service watchdog state\n"
```

Comment:
hmm, it kinda sucks that this is called "service-watchdogs" even though it applies to all services... i.e. some "service" verbs now apply to specific services, while this one applies to all them.

This really feels like we really need to clean up some naming here...

Reply:
maybe the "log-level" verbs should actually be called "daemon-log-level" or so, since we use "daemon" as a moniker for referencing the manager itself.

("daemon" is a shitty moniker for that, should probably have been "manager" i guess, but "systemctl daemon-reload" one of the primary ifaces we advertise, would be superhard to rename that)

Reply:
manager-reload sounds nice. I don't think it's a big issue to change the name: we'll keep backwards compat indefinitely.